src/main/resources/listLookup.json

```diff
@@ -358,21 +358,21 @@
         "defaultSensitivity": "",
         "parentPage": "siac-poac-paac-weekly-hearing-list"
     },
-    "FFT_TAX_WEEKLY_HEARING_LIST": {
+    "FTT_TAX_WEEKLY_HEARING_LIST": {
         "friendlyName": "First-tier Tribunal (Tax Chamber) Weekly Hearing List",
         "welshFriendlyName": "Rhestr o Wrandawiadau Wythnosol Tribiwnlys Haen Gyntaf (Siambr Treth)",
         "shortenedFriendlyName": "FFT Tax Weekly Hearing List",
-        "url": "fft-tax-weekly-hearing-list",
+        "url": "ftt-tax-weekly-hearing-list",
         "jurisdictions": ["Tribunal"],
         "welshJurisdictions": ["Tribiwnlys"],
         "restrictedProvenances": [],
         "defaultSensitivity": ""
     },
-    "FFT_LR_WEEKLY_HEARING_LIST": {
+    "FTT_LR_WEEKLY_HEARING_LIST": {
         "friendlyName": "First-tier Tribunal (Lands Registration Tribunal) Weekly Hearing List",
         "welshFriendlyName": "Rhestr o Wrandawiadau Wythnosol Tribiwnlys Haen Gyntaf (Tribiwnlys Cofrestru Tir)",
         "shortenedFriendlyName": "FFT (LR) Weekly Hearing List",
```

Comment:
The name on manual upload page still showing FFT

Reply:
Done. I was not sure about it because excel they provided have FFT and this information was shown on the interface where as all the other changes I made are relevant to backend which are not visible.